test/specs/pushAndPull.e2e.ts

```diff
@@ -10,12 +10,30 @@ import fs from 'fs';
 import { step, xstep } from 'mocha-steps';
 import path from 'path';
 import util from 'util';
-import { TestSetup } from '../testSetup';
-import * as utilities from '../utilities';
+import { TestSetup } from '../testSetup.ts';
+import * as utilities from '../utilities/index.ts';
 import { Workbench } from 'wdio-vscode-service';
 
 const exec = util.promisify(child_process.exec);
 
+async function verifyPushSuccessful(workbench: Workbench, wait = utilities.TEN_MINUTES) {
```

Comment:
Because the other function is `verifyPullSuccess()` 🙂 

```suggestion
async function verifyPushSuccess(workbench: Workbench, wait = utilities.TEN_MINUTES) {
```